src/fragments/lib/auth/flutter/advanced/advanced.mdx

```diff
@@ -0,0 +1,81 @@
+## Identity Pool Federation
+
+Imagine that you are creating a mobile app that accesses AWS resources, such as a game that runs on a mobile device and stores player and score information using Amazon S3 and DynamoDB.
+
+When you write such an app, you make requests to AWS services that must be signed with an AWS access key. However, we strongly recommend that you do not embed or distribute long-term 
```

Comment:
identity web federation? does web need to be on there?

Reply:
That's usually how it's called in AWS, e.g. https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_oidc.html

Open to using just "federation" instead

Reply:
I think I like identity federation here. let us use that.

Reply:
Done